metering/reports/metering-writing-custom-report-queries.adoc

```diff
@@ -0,0 +1,396 @@
+[id="metering-writing-custom-report-queries"]
+= Writing custom report queries
+include::modules/common-attributes.adoc[]
+:context: metering
+
+toc::[]
+
+Metering allows you to add and expand upon build-in reports and metrics by using CustomResources to gather new metrics and create new custom queries and reports. The primary CustomResources that allow this are the ReportDataSource and the ReportQuery.
+
+A ReportDataSource can configure metering to execute additional custom Prometheus queries and store the metrics collected.
+Once the data has been collected a ReportQuery can be used to analyze the metrics.
+
+This guide walks through collecting new Prometheus metrics, then writing a custom SQL query to analyze these metrics. Review the documentation on ReportDataSources and ReportDataQueries before you read this guide, or as needed while you read through.
+
+== Setup
+
+This guide assumes you have already installed metering and have Prometheus running in your cluster. 
+
+In many of the examples below you run a Prometheus query outside of the metering operator. To to this we will access Prometheus directly using the default exposed route. Run the following command to verify the route for Prometheus:
```

Comment:
s/we will access/you canaccess

Reply:
Should this be a numbered list?
. Run the following command...
. Open up your web browser ...